maestro-cli/src/main/java/maestro/cli/session/MaestroSessionManager.kt

```diff
@@ -164,18 +163,13 @@ object MaestroSessionManager {
                         val xcTestDriverClient = XCTestDriverClient(
                             host = defaultHost,
                             port = defaultXcTestPort,
-                            restoreConnection = {
-                            if (SessionStore.activeSessions().isNotEmpty()) {
```

Comment:
Note that the condition for restarting `if (SessionStore.activeSessions().isNotEmpty()) { <restart> }` is removed, and restarts now happen unconditionally.

@amanjeetsingh150 Do you happen to know why this condition is put in place? Why stop a xctest runner restart if there are no active sessions?